pkg/manager/internal/controllers/derivedcustomresource_controller.go

```diff
@@ -292,8 +292,9 @@ func (r *DerivedCustomResourceReconciler) Reconcile(req ctrl.Request) (ctrl.Resu
 		return result, nil
 	}
 
-	// leave `Paused` flag as is
+	// leave `Paused` flag and `LogLevel` as is
 	desiredElevator.Spec.Paused = currentElevator.Spec.Paused
+	desiredElevator.Spec.LogLevel = currentElevator.Spec.LogLevel
```

Comment:
This approach has a drawback, we need to find all the places that we update the Operator CRD objects, which is really hard to maintain. I think this can be solved by triggering webhook also during `UPDATE`, there is a
`req.OldObject` in the request, and you just check if the loglevel of the Oldobject is nil, if it is set, and just let it go., if it is not set, default it to operator's loglevel.

Reply:
This can also fix the issue here: https://github.com/kubermatic/kubecarrier/pull/516#discussion_r459950089

Reply:
lgtm

Reply:
I can also move `pause` flag logic also to the webhook `update` method, is it ok to do that in this PR?

Reply:
For pause flag is that ok if we do it in a dedicated PR? This PR is getting larger and larger :)

Reply:
> This approach has a drawback, we need to find all the places that we update the Operator CRD objects, which is really hard to maintain. I think this can be solved by triggering webhook also during `UPDATE`, there is a
> `req.OldObject` in the request, and you just compare the loglevel of the Oldobject with the Object, if it is same as the `OldObject`, and just let it go., if it is not set or not same, default it to OldObject loglevel.

in that way, you won't be able to change logLevel by your self